extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/IdentityHubCredentialsVerifier.java

```diff
@@ -61,41 +68,74 @@ public IdentityHubCredentialsVerifier(IdentityHubClient identityHubClient, Monit
      */
     @Override
     public Result<Map<String, Object>> getVerifiedCredentials(DidDocument didDocument) {
+        monitor.debug("Retrieving verified credentials for " + didDocument.getId());
```

Comment:
```suggestion
        monitor.debug(() -> "Retrieving verified credentials for " + didDocument.getId());
```

also in other places